1. Two_Sum/solution2.java

```diff
@@ -0,0 +1,36 @@
+public class Solution {
+    public int[] twoSum(int[] numbers, int target) {
+
+        //creating new hashmap
+        Map<Integer,Integer> map = new HashMap<>();
+
+        //running a for loop
+        for(int i=0;i< numbers.length;i++){
+
+            //making the complement of the numbers in the array
+            int compl=target-numbers[i];
+
+            //findiing a num whether it exits in hashmap
+            if (map.containsKey(compl)){
+                return new int[]{map.get(compl),i};
+                
+            }
+            //if we dont find the complement, so just put the index in hashmap
+            map.put(numbers[i],i);
+            
+        }
+        return null;
+        
+    }
+}
+/*
```

Comment:
As mentioned in the guidelines Provide Runtime, memory, time, and space complexity, author name a brief description of the solution **_at the top of the file._** A **_function should go below_** that.

So just make sure that you add it at the top, and then add the function.

Reply:
I made the suggested changes, what shall I do next?